packages/ui/components/data-table/DataTablePagination.tsx

```diff
@@ -0,0 +1,60 @@
+import type { Table } from "@tanstack/react-table";
+import { ChevronLeft, ChevronRight, ChevronsLeft, ChevronsRight } from "lucide-react";
+
+import { Button } from "../button";
+
+interface DataTablePaginationProps<TData> {
+  table: Table<TData>;
+}
+
+export function DataTablePagination<TData>({ table }: DataTablePaginationProps<TData>) {
```

Comment:
Not used here as we use infinity loading. But usefull if we ever want to use page based pagination